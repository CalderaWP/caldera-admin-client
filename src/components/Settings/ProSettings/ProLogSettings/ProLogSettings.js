import React from 'React';
import propTypes from 'prop-types';
import classNames from 'classnames'

/**
 * Create the global form settings UI
 * @param {Object} props
 * @return {*}
 * @constructor
 */
export const ProLogSettings = (props) => {
	return(
		<div
			className={classNames(props.className,ProLogSettings.classNames.wrapper)}
		>
			ProLogSettings
		</div>
	)
};

/**
 * Prop types for the GlobalForms settings component
 * @type {{}}
 */
ProLogSettings.propTypes = {
	classNames: propTypes.string
};

/**
 * Default props for the GlobalForms settings component
 * @type {{}}
 */
ProLogSettings.defaultProps = {

}

/**
 * Class names used in the GlobalForms settings component
 * @type {{wrapper: string}}
 */
ProLogSettings.classNames = {
	wrapper: 'caldera-forms-global-form-settings'
}