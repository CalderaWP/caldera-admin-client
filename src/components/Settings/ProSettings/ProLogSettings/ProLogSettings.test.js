import renderer from 'react-test-renderer';
import React from 'react';
import { shallow } from 'enzyme';
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import {ProLogSettings} from "./ProLogSettings";

Enzyme.configure({ adapter: new Adapter() });

describe( 'ProLogSettings component', () => {
	it( 'Matches snapshot with minimal props', () => {
		expect(
			renderer.create(
				<ProLogSettings/>
			).toJSON()
		).toMatchSnapshot()
	});

	it( 'Is wrapped in the right class', () => {
		expect(
			shallow(
				<ProLogSettings/>
			).find( '.' + ProLogSettings.classNames.wrapper )
				.length
		).toEqual(1)
	});
});