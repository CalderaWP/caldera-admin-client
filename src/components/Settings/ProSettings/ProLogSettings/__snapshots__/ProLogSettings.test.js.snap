// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProLogSettings component Matches snapshot with minimal props 1`] = `
<div
  className="caldera-forms-global-form-settings"
>
  ProLogSettings
</div>
`;
